block/manager.go

```diff
@@ -36,13 +39,14 @@ type Manager struct {
 
 	dalc      da.DataAvailabilityLayerClient
 	retriever da.BlockRetriever
+	daHeight  uint64 // height of the latest processed DA block
 
 	HeaderOutCh chan *types.Header
 	HeaderInCh  chan *types.Header
 
 	syncTarget uint64
 	blockInCh  chan *types.Block
-	retrieveCh chan uint64
+	retrieveCh chan interface{}
```

Comment:
It is a bit unclear what the purpose of the `retrieveCh` is imo. Maybe add a comment.

Reply:
Actually, I think I'm going to replace this one with different synchronization method.